maat-court-data-api/src/main/java/gov/uk/courtdata/assessment/controller/FinancialAssessmentController.java

```diff
@@ -91,11 +97,20 @@ public ResponseEntity<OutstandingAssessmentResultDTO> checkForOutstandingAssessm
     @PostMapping(value = "/history/{financialAssessmentId}/fullAvailable/{fullAvailable}", produces = MediaType.APPLICATION_JSON_VALUE)
     @Operation(description = "Create financial assessment, details and child weight history record")
     @ApiResponse(responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE))
-    @ApiResponse(responseCode = "400", description = "Bad Request.", content = @Content(mediaType = "application/json", schema = @Schema(implementation = ErrorDTO.class)))
-    @ApiResponse(responseCode = "500", description = "Server Error.", content = @Content(mediaType = "application/json", schema = @Schema(implementation = ErrorDTO.class)))
+    @StandardApiResponse
     public ResponseEntity<Object> createAssessmentHistory(@PathVariable int financialAssessmentId, @PathVariable boolean fullAvailable) {
         log.info("Create Assessment History Request Received");
         financialAssessmentHistoryService.createAssessmentHistory(financialAssessmentId, fullAvailable);
         return ResponseEntity.ok().build();
     }
+
+    @GetMapping(value = "/{financialAssessmentId}/ioj-assessor-details", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
please change ioj-assessor-details to means-assessor-details as it is part of the means assessment

Reply:
agreed, done as below

```
    @GetMapping(value = "/{financialAssessmentId}/means-assessor-details", produces = MediaType.APPLICATION_JSON_VALUE)
    @Operation(description = "Retrieve details of the interests of justice assessor for a given financial assessment")
    @ApiResponse(responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE))
    @StandardApiResponse
    @NotFoundApiResponse
    public ResponseEntity<AssessorDetails> findMeansAssessorDetails(@PathVariable int financialAssessmentId) {
        AssessorDetails meansAssessorDetails = financialAssessmentService.findMeansAssessorDetails(financialAssessmentId);
        return ResponseEntity.ok(meansAssessorDetails);
    }
```